### DISCUSSION TITLE
今天安卓手机突然上不了网了

### DISCUSSION THREAD

**Discussion by tearlake** [2dust/v2rayNG#4076] [Q&A]
昨天还没问题  但今天启动v2rayNG后就无法上网了  同样的节点在电脑上可以使用  手机上测试真连接也通   卸载后退回几个版本再安装也不行  请问是什么情况

**Comment by woshichenghaibo** [comment]
我跟你一样的情况

我的是在一个服务器上弄的节点在手机上从来没有联通过，在pc上都正常，在手机上可以测试真链接延迟但是就是上不去网，我到处打听也找不到原因，你知道日志文件在哪里找么

**Reply by pcorvoh** [reply]
我现在也是这样，不知道为啥，你找到了解决方案吗？

**Comment by elarbor** [comment]
这个仓库是安卓手机的vpn吗？


那https://github.com/2dust/v2flyNG 仓库是干啥的？
